packages/orbit-components/src/ServiceLogo/ServiceLogo.ct.tsx

```diff
@@ -2,12 +2,27 @@ import * as React from "react";
 import { test, expect } from "@playwright/experimental-ct-react";
 
 import ServiceLogoStory from "./ServiceLogo.ct-story";
+import { NAME_OPTIONS, SIZE_OPTIONS } from "./consts";
+
+const GRAYSCALE_OPTIONS_SIZE = 2;
 
 test.describe("visual ServiceLogo", () => {
   test("default", async ({ mount, page }) => {
     const component = await mount(<ServiceLogoStory />);
     await page.waitForRequest(/images.kiwi.com/);
 
-    await expect(component).toHaveScreenshot();
+    const selector = `img:nth-of-type(${
+      Object.values(NAME_OPTIONS).length +
+      Object.values(SIZE_OPTIONS).length +
+      GRAYSCALE_OPTIONS_SIZE
+    })`;
+
+    await page.waitForSelector(selector);
+
+    await expect(component).toHaveScreenshot({
+      threshold: 0.3,
+      maxDiffPixelRatio: 0.02,
+      maxDiffPixels: undefined,
```

Comment:
How does it behave when this value is undefined?

I see that [this value is unset by default](https://playwright.dev/docs/api/class-pageassertions#page-assertions-to-have-screenshot-1-option-max-diff-pixels), so I assume it would be the same as not setting it at all?

Reply:
This is `undefined` here so this overrides the [global config](https://github.com/kiwicom/orbit/blob/23532b846c399467f722c85aff8f7c048f815db6/packages/orbit-components/playwright-ct.config.ts#L19). We don't want this to be used on this test.